scripts/openyurt-deployer/README.md

```diff
@@ -0,0 +1,153 @@
+# Quick set-up `OpenYurt`
+
```

Comment:
add an overview of the Knative-atop-OpenYurt mode. You can give the architecture overview figure (from our tutorial slides) and a pointer to the tutorial slides.